scripts/release.sh

```diff
@@ -24,12 +24,20 @@ npm run system-test
 git config --global user.name "travis-ci"
 git config --global user.email "travis@travis-ci.org"
 
+if [[ ${TRAVIS_TAG} == *"-"* ]]; then
```

Comment:
@stephenplusplus so based on what we said about tagging, WDYT about these changes?